src/ua/griddynamics/Game.java

```diff
@@ -0,0 +1,207 @@
+package ua.griddynamics;
+
+import java.util.Random;
+
+public class Game {
+    static final int WIDTH = 9;
+    static final int HEIGHT = 9;
+
+    public final char X = 'X';
+    public final char FREE = '/';
+    public final char MINE = '*';
+    public final char POINT = '.';
+    public final char ZERO = '0';
+
+    private final int numberOfMines;
+    private final Cell[][] fields;
+
+    public Game(int numberOfMines) {
+        this.fields = new Cell[WIDTH][HEIGHT];
+        for (int i = 0; i < fields.length; i++) {
+            for (int j = 0; j < fields[i].length; j++) {
+                fields[i][j] = new Cell();
+            }
+        }
+        this.numberOfMines = numberOfMines;
+    }
+
+    public void placeMines(Coordinate coordinate) {
+        Random random = new Random();
+        int minesPlaced = 0;
+        int userX = coordinate.getX();
+        int userY = coordinate.getY();
+        do {
```

Comment:
It is better to use while instead of do/while. It is easier to read.
Also e.g. for this case the logic is not correct if numberOfMines=0